src/ralph/admin/widgets.py

```diff
@@ -155,31 +154,32 @@ def get_related_url(self, info, action, *args):
             args=args
         )
 
+    def value_from_datadict(self, data, files, name):
+        value = data.get(name)
+        if getattr(self, 'multi', False) is True:
+            print('name', name)
+            if value:
+                value = value.split(',')
+            else:
+                value = []
+        return value
+
     def render(self, name, value, attrs=None):
-        admin_model = self.admin_site._registry[self.rel_to]
         model_options = (
             self.rel_to._meta.app_label, self.rel_to._meta.model_name
         )
-        widget_options = {
-            'data-suggest-url': reverse(
-                'autocomplete-list', kwargs={
-                    'app': self.field.rel.related_model._meta.app_label,
-                    'model': self.field.rel.related_model._meta.model_name,
-                    'field': self.field.name
-                }
-            ),
-            'data-details-url': reverse(
-                'admin:{}_{}_autocomplete_details'.format(*model_options)
-            ),
-            'data-query-var': QUERY_PARAM,
-            'data-detail-var': DETAIL_PARAM,
-        }
-
         if attrs is None:
             attrs = {}
-        attrs.update({'type': 'hidden'})
-        widget_options['data-target-selector'] = '#' + attrs.get('id')
-        input_field = super().render(name, value, attrs)
+        attrs['name'] = name
+        if self.multi:
+            attrs['multi'] = ''  # only key is requied
+            value = ','.join(force_text(v) for v in value)
```

Comment:
it ended up, it's correct solution. I have only replaced ',' with self.separator.